cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/ResourcesJdbcRepositoryIT.java

```diff
@@ -0,0 +1,61 @@
+package com.github.missioncriticalcloud.cosmic.api.usage.repositories.jdbc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.github.missioncriticalcloud.cosmic.usage.core.model.PublicIp;
+import com.github.missioncriticalcloud.cosmic.usage.core.model.VirtualMachine;
+import com.github.missioncriticalcloud.cosmic.usage.core.model.Volume;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("local")
+public class ResourcesJdbcRepositoryIT {
```

Comment:
I'd suggest having individual tests for each repository. That means splitting `ResourcesJdbcRepositoryIT` into `VirtualMachinesJdbcRepositoryIT`, `VolumesJdbcRepositoryIT`, etc.